assets/js/components/notifications/BannerNotifications.js

```diff
@@ -84,9 +85,12 @@ export default function BannerNotifications() {
 					{ isAuthenticated && <CoreSiteBannerNotifications /> }
 					{ dashboardSharingEnabled && <ModuleRecoveryAlert /> }
 					{ ga4ActivationBannerEnabled && <ActivationBanner /> }
-					{ gteSupportEnabled &&
-						ga4ModuleConnected &&
-						hasGTMScope && <GoogleTagIDMismatchNotification /> }
+					{ gteSupportEnabled && ga4ModuleConnected && hasGTMScope && (
+						<Fragment>
+							<GoogleTagIDMismatchNotification />
+							<WebDataStreamNotAvailableNotification />
```

Comment:
⚠️ This placement seems incorrect because the component doesn't depend on GTE or the GTM scope. It should however be only shown to authenticated users since it uses module settings which won't be available in a view-only context.

Reply:
After further review, it seems okay since the code that triggers it only happens when syncing Google tag settings so while this seems off at first glance, it should be fine.